frontend/routes/package/(_components)/Docs.tsx

```diff
@@ -83,7 +83,7 @@ export function DocsView({ docs, params, selectedVersion }: DocsProps) {
           version={selectedVersion.version}
         />
         <div
-          class="ddoc w-full lg:min-h-0 lg:*:!h-full"
+          class="ddoc w-full lg:*:max-h-[calc(100vh-75px)]"
```

Comment:
`75px` is not a random number. `100vh-75px` is the correct height of scrollable area so when sidebar sticks to the top it fills the whole page. The [container](https://github.com/jsr-io/jsr/blob/73bd3a20d647485663c25a15c03c65a389a9f7ba/frontend/routes/package/(_components)/Docs.tsx#L79) has `py-4` and the [search input](https://github.com/jsr-io/jsr/blob/73bd3a20d647485663c25a15c03c65a389a9f7ba/frontend/routes/package/(_islands)/LocalSymbolSearch.tsx#L165) has `35px` of height and `m-2` which all sum up to `75px`.